via/views/exceptions.py

```diff
@@ -79,8 +74,7 @@ def all_exceptions(exc, request):
     try:
         status_code = exc.status_int
     except AttributeError:
-        # This is our 501, chosen to not scare Cloudflare.
```

Comment:
What was the "don't scare Cloudflare" scenario that was a concern here? Is it still relevant in any way?

Reply:
I don't think this comment was every correct. 501 is 501 Not Implemented. The comment must have meant 502 Gateway Error. In the past people have sometimes thought that, when a request to a third-party fails, it would be semantically correct for us to return 502 Gateway Error. But this has been found to cause a problem in the past: Cloudflare replaces 502 response from us with their own Gateway Error page. That must be what this comment referred to. But the `except` block that contains this comment is never entered when handling an exception related to an upstream request, so the comment makes no sense here.